code/game/objects/structures/crates_lockers/closets/statue.dm

```diff
@@ -82,10 +82,12 @@
 	var/obj/structure/statue/petrified/S = new(loc, src, statue_timer)
 	S.name = "statue of [name]"
 	bleedsuppress = 1
-	S.icon = icon
-	S.icon_state = icon_state
+	if(dna.species.id == "lizard")
+		S.icon_state = "lizard"
+	else
+		S.icon_state = "human_male"
 	S.copy_overlays(overlays)
```

Comment:
Just tested this -- it does produce a human sprite but it does not copy mutant bodyparts, so lizards would still look wrong